pkg/relay/ethereum/events.go

```diff
@@ -40,15 +42,20 @@ func init() {
 }
 
 func (chain *Chain) findSentPackets(ctx core.QueryContext, fromHeight uint64) (core.PacketInfoList, error) {
+	logger := chain.GetChannelLogger()
+	defer logger.TimeTrack(time.Now(), "findSentPackets")
 	var dstPortID, dstChannelID string
 	if channel, found, err := chain.ibcHandler.GetChannel(
 		chain.callOptsFromQueryContext(ctx),
 		chain.Path().PortID,
 		chain.Path().ChannelID,
 	); err != nil {
+		logger.Error("failed to get channel", err)
 		return nil, err
 	} else if !found {
-		return nil, fmt.Errorf("channel not found: sourcePortID=%v sourceChannel=%v", chain.Path().PortID, chain.Path().ChannelID)
+		err := fmt.Errorf("channel not found")
+		logger.Error("failed to get channel", err, "sourcePortID", chain.Path().PortID, "sourceChannel", chain.Path().ChannelID)
```

Comment:
@dongrie thank you but change some keys.
- "sourcePortID" => "port_id"
- "sourceChannel" => "channel_id"

`chain` is not necessarily the source chain, so the original keys are confusing.

Reply:
https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/13/commits/4611488cb01d764157417b2ccbf882ec9e47d78f
Changed key name